packages/svelteui-core/src/lib/components/Timeline/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Timeline } from './combinator';
+export * as TimelineStyles from './Timeline.styles';
+export * as TimelineItemStyles from './TimelineItem/TimelineItem.styles';
```

Comment:
I'll fix it for you, but we're moving toward exporting the types directly now. Just a note for next time.